src/view/configuration/configuration.jsx

```diff
@@ -150,7 +150,20 @@ const validationSchema = object()
   .shape({
     instances: array().of(
       object().shape({
-        name: string().required("Please specify a name."),
+        name: string()
+          .required("Please specify a name.")
+          // Under strict mode, setting window["123"], where the key is all
+          // digits, throws a "Failed to set an indexed property on 'Window'" error.
+          // This regex ensure there's at least one non-digit.
```

Comment:
Good catch! Should we add this to the docs?

Reply:
Yes. Done! https://github.com/AdobeDocs/alloy-docs/pull/54